app/Resources/FOSUserBundle/views/layout.html.twig

```diff
@@ -1,26 +1,26 @@
 {% extends "::base.html.twig" %}
 
 {% block stylesheets %}
-{{ parent() }}
-<style>
-#loading {
-    display: none;
-}
-</style>
+    {{ parent() }}
+    <style>
+    #loading {
+        display: none;
+    }
+    </style>
 {% endblock %}
 
 {% block sidebar %}
-{% if app.user %}
+{# {% if app.user %}
```

Comment:
This is because we hide the sidebar, right? If so, just delete everything that you've commented.